dotnet-desktop-guide/net/wpf/documents/how-to-print-xps-files.md

```diff
@@ -1,28 +1,59 @@
 ---
-title: How to display a message box
-description: 
-ms.date: 
-dev_langs: 
+title: How to print an XPS file
+description: Learn how to print an XPS file from your application by using the System.Printing.PrintQueue.AddJob method.
+ms.date: 09/16/2021
+dev_langs:
   - "csharp"
   - "vb"
 helpviewer_keywords:
-  - 
+  - "programmatically print xps files [WPF]"
 ---
 
 # How to print an XPS file (WPF .NET)
 
-<!--
+Sometimes you'll want to add a new print job to the print queue without opening a print dialog box. You can use one of the [PrintQueue.AddJob](/dotnet/api/system.printing.printqueue.addjob) methods to do that. Here's how.
```

Comment:
Fix this link to xref 😄 and the others in the article too

Reply:
Done.